ibu-imager/Dockerfile

```diff
@@ -13,7 +13,7 @@ COPY vendor/ vendor/
 # Copy the Go source installation_configuration_files
 COPY main.go main.go
 COPY cmd/ cmd/
-COPY internal/ internal/
+COPY pkg/ internal/
```

Comment:
I assume we could just drop this dockerfile?

Reply:
yep, though to do in the next pr